.github/workflows/circleci_redirect.yml

```diff
@@ -0,0 +1,23 @@
+# To enable this workflow on a fork, comment out:
+#
+# if: github.repository == 'numpy/numpy-financial'
```

Comment:
I don't see this anywhere in the workflow, did you delete that line to simplify it?